core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -329,6 +329,11 @@ public enum Property {
           + " was changed and it now can accept multiple class names. The metrics spi was introduced in 2.1.3,"
           + " the deprecated factory is org.apache.accumulo.core.metrics.MeterRegistryFactory.",
       "2.1.0"),
+  GENERAL_SERVER_LOCK_VERIFICATION_INTERVAL("general.server.lock.verification.interval", "2m",
```

Comment:
Wondering if this should be disabled by default in 2.1.4.  Enabled by default in 3.1.0.  If its working well could enable by default in 2.1.5.  If this code has a bug that causes a false positive it could bring a cluster down.

Reply:
Yeah, can make that change on the merge up to 3.1

Reply:
Set to zero in 601aff5.